react/hooks/useBulkImportsQuery.ts

```diff
@@ -0,0 +1,21 @@
+import useSWR from 'swr'
+
+import type { Session } from '../modules/session'
+import { useSessionResponse } from '../modules/session'
+import { getBulkImportList } from '../services'
+
+const useBulkImportQuery = () => {
+  const session = useSessionResponse() as Session
+
+  const account = session?.namespaces?.account?.accountName?.value
+
+  return useSWR(
+    account ? 'bulk-import-list' : null,
+    () => getBulkImportList(account),
+    {
+      refreshInterval: 30 * 1000, // 30 seconds
```

Comment:
out of curiosity: what is this refreshInterval for exactly? Do we need to keep calling the bulk import list every 30 seconds?

Reply:
Yep, that's what this line would do . I imagine that this is to test the polling feature of SWR. We'll need to do polling when an import is in progress, so that we can update the progress status bar. I believe we can skip polling when there's no import in progress though, so this refreshInterval is something that we'll probably be setting (and unsetting) dynamically, right @ArthurTriis1 ?

Reply:
Yes, that's right! I can stop the interval if that is not in progress bulk import and trigger when a new bulk import is uploaded